### DISCUSSION TITLE
Release plans for Fluent v9 List Component (Preview)

### DISCUSSION THREAD

**Discussion by mittal-sarthak** [microsoft/fluentui#33333] [General] [answered]
Hey Team,
I just wanted to check if there is any timeline available on the release/ GA for the List component in Fluent v9. Would like to use this for one of our use case and like to get a idea if this is planned to get released in near future. 
Thanks 
https://react.fluentui.dev/?path=/docs/preview-components-list--docs

**Comment by spmonahan** [comment]
@gouttierre Can you share a timeline for this control?

**Reply by spmonahan** [reply, marked as answer]
Looks like it should be soon: #33378
